jaxrs/src/main/java/org/jboss/as/jaxrs/deployment/JaxrsSpringProcessor.java

```diff
@@ -134,6 +129,26 @@ public Closeable getValue() throws IllegalStateException, IllegalArgumentExcepti
         }
     }
 
+    private void installMountHandleService(final File file,final VirtualFile vf) throws IOException {
+        final Closeable mountHandle = VFS.mountZip(file, vf, TempFileProviderService.provider());
+        Service<Closeable> mountHandleService = new Service<Closeable>() {
+            public void start(StartContext startContext) throws StartException {
+                JaxrsLogger.JAXRS_LOGGER.warn("Service for managing handle on " + vf.getName() + " is starting");
```

Comment:
This should probably be a debug statement. Also it could use `debugf("Service for managing handle on %s is starting", vf.getName())`.